bluebottle/time_based/tests/test_admin.py

```diff
@@ -73,9 +73,26 @@ def test_list_activities_office(self):
         response = self.app.get(url)
         self.assertEqual(len(response.html.find_all("a", string="Schin op Geul")), 2)
 
+    def test_participant_count(self):
+        activity = DateActivityFactory.create()
+        DateParticipantFactory.create_batch(5, activity=activity)
+        url = reverse('admin:time_based_dateactivity_changelist')
+        response = self.app.get(url)
+        self.assertEqual(int(response.html.find_all('td')[5].get_text()), 5)
 
-class DateActivityAdminScenarioTestCase(BluebottleAdminTestCase):
+    def test_start_date(self):
+        activity = DateActivityFactory.create()
+        DateActivitySlotFactory.create_batch(2, activity=activity)
+        start_time = activity.slots.all()[0].start.astimezone(get_current_timezone())
```

Comment:
This looks a little fragile to me. The DateActivitySlot model has an ordering of `['start', 'id']`, so by default the slots are ordered by start date. But this is kind of an implementation detail, which is why we sort the slots explicitly in the admin property. Either add a comment that this test works because of this ordering, or explicitly find the earliest slot.